### ISSUE TITLE
Insiders Feature: Send ephemeral messages

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1049]

**Comment by smashah** [comment]
After investigating. It looks like an individual message cannot be ephemeral, the whole group chat setting must be set to ephemeral.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.1.3](https://github.com/open-wa/wa-automate-nodejs/compare/3.1.2...3.1.3) (2020-12-04)

- build(deps): bump ts-json-schema-generator from 0.79.0 to 0.80.0 [`#1130`](https://github.com/open-wa/wa-automate-nodejs/pull/1130)
- build(deps): bump change-case from 4.1.1 to 4.1.2 [`#1131`](https://github.com/open-wa/wa-automate-nodejs/pull/1131)
- build(deps-dev): bump ts-node from 9.0.0 to 9.1.0 [`#1135`](https://github.com/open-wa/wa-automate-nodejs/pull/1135)
- patch: fixes #1132 [`#1132`](https://github.com/open-wa/wa-automate-nodejs/issues/1132)
- Release 3.1.3 [`4bf1c68`](https://github.com/open-wa/wa-automate-nodejs/commit/4bf1c6814cb6c16581205d57654929e32251870a)
- update docs email addresses [`4b3fc35`](https://github.com/open-wa/wa-automate-nodejs/commit/4b3fc350c8cba1f52da93ea8efaf500e8e8f31ab)
- Insiders: Now able to mention users in file captions [patch] #1068 [`963972b`](https://github.com/open-wa/wa-automate-nodejs/commit/963972b0beb28d0dcb351bc766bfd6460c6ccfdb)
- Insiders: Implemented `setChatEphemeral` #1049 [`b233389`](https://github.com/open-wa/wa-automate-nodejs/commit/b233389047289e8f292eb9ff7ee5b7e8e43b1f5f)
- added random hash timestamp to host notification. `getMessageReaders` now returns property `t` representing the time the messsage was read [`8219e4c`](https://github.com/open-wa/wa-automate-nodejs/commit/8219e4ce1b054ed7763b7e80c9fae45c881e5bb2)
- allow `--browser-revision` to be set via CLI #1124 [`e244778`](https://github.com/open-wa/wa-automate-nodejs/commit/e2447781506990a0d5bf1c88780ef5b9a2e02d30)

**Comment by smashah** [comment]
https://docs.openwa.dev/classes/client.html#setchatephemeral

```
await client.setChatEphemeral(chatId, true/false)
```
